packages/api-docs/config/api-docs.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
     path.resolve(__dirname, '../../react-link/dist/react-link.api.json'),
     path.resolve(__dirname, '../../react-slider/dist/react-slider.api.json'),
     path.resolve(__dirname, '../../react-tabs/dist/react-tabs.api.json'),
+    path.resolve(__dirname, '../../react-theme-provider/dist/react-theme-provider.api.json'),
```

Comment:
Forgot to mention, this should be added as a dependency of api-docs to ensure things build in the right order. (that will probably already work due to it being an indirect dep, but good to be explicit)

Reply:
thanks for pointing out; added missing deps